narwhals/group_by.py

```diff
@@ -30,9 +31,7 @@ def __init__(self: Self, df: DataFrameT, *keys: str, drop_null_keys: bool) -> No
             *self._keys, drop_null_keys=drop_null_keys
         )
 
-    def agg(
-        self: Self, *aggs: IntoExpr | Iterable[IntoExpr], **named_aggs: IntoExpr
-    ) -> DataFrameT:
+    def agg(self: Self, *aggs: Expr | Iterable[Expr], **named_aggs: Expr) -> DataFrameT:
```

Comment:
I think this should be:

```suggestion
    def agg(self: Self, *aggs: Expr, **named_aggs: Expr) -> DataFrameT:
```
as we don't expect someone to pass `.agg( (nw.sum("a"), nw.sum("b")), (nw.mean("a"), ))`

Reply:
yeah it's allowed to do `.agg([nw.sum('a'), nw.sum('b')])`